app/src/main/java/org/wikipedia/database/DatabaseTable.kt

```diff
@@ -98,6 +98,6 @@ abstract class DatabaseTable<T>(private val tableName: String, val baseContentUR
     }
 
     companion object {
-        protected const val INITIAL_DB_VERSION = 1
```

Comment:
Not sure why I had to take out this protection... since PageImageDatabaseTable is inheriting from DzatabaseTable..